static/js/services/playsound.js

```diff
@@ -156,7 +156,65 @@ define(['underscore', 'Howler', 'require'], function(_, Howler, require) {
 	window.PLAYSOUND = registry; // make available for debug.
 
 	// playSound
-	return [function() {
+	return ['translation', 'appData', 'mediaStream', function(translation, appData, mediaStream) {
+
+		/*
+		  Api
+		  0: Default (Play sound)
+		  1: No sounds when in call
+		  2: No sounds
+		*/
+		var scope;
+		var playSoundDefaults = {
+			join: 0,
+			chat: 0,
+			call: 0
+		}
```

Comment:
Please use named constants for the different types, this makes the comparisons below more readable..